cert-manager/issuer.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   acme:
     server: https://acme-v02.api.letsencrypt.org/directory
     # server: https://acme-staging-v02.api.letsencrypt.org/directory
-    email: viktor@farcic.com
+    email: wiggitywhitney@gmail.com
```

Comment:
This change is OK since it does not do any "damage". However, later on, if you follow the instructions, you will be adding and changing "stuff" that should not be merged to the mainline since it greatly depends on the path one takes. Normally, I create a separate branch for "testing" or "playing" with the "adventure" and do not merge it to the mainline and make changes to the mainline only when they are intentionally meant to be there and visible from the beginning of the "adventure".

Reply:
Heard!  I was going to ask about this!